### DISCUSSION TITLE
Algorithm parameter for running tumor-only samples

### DISCUSSION THREAD

**Discussion by rachelmadang** [uclahs-cds/metapipeline-DNA#83] [Q&A] [answered]
Will metapipeline recognize that samples are tumor only and use appropriate algorithm for call-sSNV since not all of them support tumor only samples or is that something I need to specify in the config file?

**Comment by yashpatel6** [comment, marked as answer]
Yes the metapipeline will handle selecting the appropriate algorithm for call-sSNV based on the input type from the CSV/YAML and sample mode from the config. Ex. tumor only, single sample mode --> metapipeline will only run the sSNV algorithms that support that input.
